### DISCUSSION TITLE
View local changes and history in one window

### DISCUSSION THREAD

**Discussion by wl2776** [sourcegit-scm/sourcegit#763] [Q&A] [answered]
Is it possible?

Currently I have to switch between "History" and "Local changes" in top left block.

Here is how it is implemented in SmartGit

![image](https://github.com/user-attachments/assets/bcb23a24-844d-43d4-b614-b95be0d2a3f0)

**Comment by love-linger** [comment, marked as answer]
There are currently no plans to add this feature. When working on local changes, I would like more space to display the list of changes both staged and unstaged, and the detail changes of selected file to make it easier to review the code.
